src/Centreon/Application/Controller/MonitoringResourceController.php

```diff
@@ -193,7 +193,9 @@ public function list(
                 $parameters['serviceId'] = $resource->getId();
             }
 
-            $resource->setAcknowledgementEndpoint($this->router->generate($routeNameAcknowledgement, $parameters));
+            $resource->setAcknowledgementEndpoint(
+                $this->router->generate($routeNameAcknowledgement, array_merge($parameters, ['limit' => 1]))
```

Comment:
this allows to get only the last acknowledgement of the resource
It makes no sense to display several acknowledgements